### PR TITLE
Update CI 

### PR DESCRIPTION
**Pull request by vsreekanti**

### CODE REVIEW

**Comment by vsreekanti** [comment]
Test this please.

**Comment by vsreekanti** [comment]
Test this please.

**Comment by shaneknapp** [comment]
test this please

File: ci/Dockerfile [hunk 1]
```diff
@@ -54,8 +54,10 @@ RUN sed -i "s|#dbms.shell|dbms.shell|g" /etc/neo4j/neo4j.conf
 # install git
 RUN apt-get install -y git
 
+ADD ground/ /tmp/ground
```
Comment (shaneknapp) [inline ci/Dockerfile]: change this to:
ADD ./ /tmp/ground

File: ci/Dockerfile [hunk 2]
```diff
@@ -54,8 +54,10 @@ RUN sed -i "s|#dbms.shell|dbms.shell|g" /etc/neo4j/neo4j.conf
...
+
 # add test running script for CMD
-ADD start_servers_and_test.sh /tmp/
+ADD ground/ci/start_servers_and_test.sh /tmp/
```
Comment (shaneknapp) [inline ci/Dockerfile]: change this to:
ADD ./ci/start_servers_and_test.sh /tmp/

**Review by shaneknapp — CHANGES_REQUESTED** [review]
i've been testing on our staging worker, and got the docker container build working w/these changes.  your tests are failing, but that could just be from an old commit.

File: ci/start_servers_and_test.sh [hunk 3]
```diff
@@ -41,7 +31,7 @@ mvn clean test
 
 # generate the test coverage report and send it to Coveralls only if we're
 # building on the master branch
-if [ $PULL_PATH == "master" ] 
+if [ -z "$MASTER_BUILD" ]
```
Comment (shaneknapp) [inline ci/start_servers_and_test.sh]: change to:
if [ ! -z "${MASTER_BUILD}" ]

**Pull request merged**